features/camel/src/main/feature/feature.xml

```diff
@@ -16,285 +16,434 @@
           xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
-    <repository>mvn:org.apache.karaf.features/spring/${karaf.version}/xml/features</repository>
-    <repository>mvn:ddf.features/cxf/${project.version}/xml/features</repository>
-    <repository>mvn:org.apache.karaf.features/enterprise/${karaf.version}/xml/features</repository>
+  <repository>mvn:ddf.features/cxf/${project.version}/xml/features</repository>
 
-    <!--START Camel Features-->
-    <!-- DO NOT EDIT
-        All feature definitions come from 3rd party feature file:
-        <repository>mvn:org.apache.camel.karaf/apache-camel/2.24.2/xml/features</repository>
+  <!--START Camel Features-->
+  <!-- DO NOT EDIT
+      All feature definitions come from 3rd party feature file:
+      <repository>mvn:org.apache.camel.karaf/apache-camel/3.4.4/xml/features</repository>
```

Comment:
👍 I reviewed the diff with https://github.com/apache/camel-karaf/blob/camel-karaf-3.4.4/platforms/karaf/features/src/main/resources/features.xml. Everything is exactly the same besides what's mentioned in the DDF changes made list.